Tests/DependencyInjection/Compiler/CommandHandlerPassTest.php

```diff
@@ -29,16 +29,12 @@ protected function setUp()
 
     public function testProcess()
     {
-        $definition = \Mockery::mock(Definition::class);
-
-        $this->configShouldBe(
+        $this->parametersShouldBe(
```

Comment:
because they were wrong. They supposed that `getExtensionConfig` returns processed config, it does not in fact.

It is the heart of this PR. We cannot relay on `getExtensionConfig` to get processed config, so we inject stuff from bundle extension into parameters to be able to read it into CompilerPass